src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsEmptyValues.java

```diff
@@ -125,6 +134,24 @@ Object returnValueFor(Class<?> type) {
             return JavaEightUtil.emptyIntStream();
         } else if ("java.util.stream.LongStream".equals(type.getName())) {
             return JavaEightUtil.emptyLongStream();
+        } else if ("java.time.Duration".equals(type.getName())) {
+            return JavaEightUtil.emptyDuration();
+        } else if ("java.time.Period".equals(type.getName())) {
+            return JavaEightUtil.emptyPeriod();
+        } else if ("java.time.Instant".equals(type.getName())) {
+            return JavaEightUtil.emptyInstant();
+        } else if ("java.time.LocalDate".equals(type.getName())) {
+            return JavaEightUtil.emptyLocalDate();
+        } else if ("java.time.LocalDateTime".equals(type.getName())) {
+            return JavaEightUtil.emptyLocalDateTime();
+        } else if ("java.time.LocalTime".equals(type.getName())) {
+            return JavaEightUtil.emptyLocalTime();
+        } else if ("java.time.OffsetDateTime".equals(type.getName())) {
+            return JavaEightUtil.emptyOffsetDateTime();
+        } else if ("java.time.OffsetTime".equals(type.getName())) {
+            return JavaEightUtil.emptyOffsetTime();
+        } else if ("java.time.ZonedDateTime".equals(type.getName())) {
+            return JavaEightUtil.emptyZonedDateTime();
         }
```

Comment:
Do we want a default for `java.time.ZoneId` (`ZoneOffset.UTC`) and `java.time.ZoneOffset` (same)?

Reply:
fyi ZonedDateTime is already coded ;) I can do the rest if you think its needed

Reply:
It's tricky to figure out where to draw the line. I can see an argument for just doing `Duration` and `Period`, given that there is really no "empty" Instant, LocalDate, LocalTime, etc (the unix epoch is rather arbitrary).

That being said, I'd suggest either doing *only* `Duration` and `Period`, or doing all of the `java.time` types. Up to you.

Reply:
ok, it was tough decision to delete all the code I already made, but I agree that it's hard to define what an empty Instant, LocalDate etc. is. I left only period and duration. the rest is in git history if some day you will want to restore it ;)

Reply:
Thanks - I understand it's a tough decision, but it's always easier to add than to remove :-)